src/main/java/food/finder/foodfinder/user/User.java

```diff
@@ -22,11 +23,14 @@
     @Property
     private final String username;
 
+    @Property
+    private Date createDate;
+
     @Relationship(type = "LIKED", direction = Relationship.Direction.OUTGOING)
     private Set<Place> likedPlaces = new HashSet<>();
 
     public User(String email, String username) {
         this.email = email;
         this.username = username;
     }
```

Comment:
Should createdDate be defaulted here?

Reply:
Good question I think that it ca. be null but can't be saved if null.